+exp/Contents.m

```diff
@@ -0,0 +1,53 @@
+% +EXP Classes and functions for the Rigbox Experiment framework
+%   The Experiment framework provides a number of features for stimuli-
```

Comment:
I would change this opening paragraph to something like "Rigbox's Experiment framework is a a framework for implementing trial-structured behavioral tasks which involve stimuli presentation. The framework allows for parameterizing  individual experiments at a single-trial level. Rigbox allows for implementing multiple experiment frameworks - the default and recommended is the Signals framework (see `exp.SignalsExp` and/or link to signals repo)"

I think if you can parameterize trials, then parameterizing entire tasks is self-evident.